### ISSUE TITLE
Inconsistent return type for errors

### ISSUE THREAD

**Issue by jewzaam** [lightblue-platform/lightblue-core#158]
Related to #149 

```
Please note that a similar problem exists when lightblue returns errors.

For multiple errors response.data.errors is an array of error objects.
For a single error response.data is an error object.
```

**Comment by luan-cestari** [comment]
Moved to in_progress as I can't see any change from the last comment from @paterczm so I think it was in peding pr merge by msitake

**Comment by bserdar** [comment]
This was fixed with rest #26. Closing.
